src/pages/program.jsx

```diff
@@ -575,7 +575,7 @@ const Program = () => {
               <ProgramItem
                 dayStr="2022-09-16"
                 startTime="20:00"
-                endTime="22:00"
+                endTime="23:00"
```

Comment:
Shall we have the outdoor set till 11pm and then move inside?

Reply:
Wasn't it that we are allowed until 22?

Reply:
Yes but there is a grace period ;)